success-criteria.md

```diff
@@ -337,21 +309,23 @@ stakeholder group.
 
         -   Free elections protected against foreign manipulation
 
-            -   While free speech requires allowing speech the majority does not
-                approve of, we can label political speech by the author’s
-                nationality and whether it is endorsed by one or more candidates
-
-        -   Freedom of the press enables watch-dog reporting on important issues
-            and combats fake news
-
     -   Freedom of information to provide fast, easy access to internet-enabled
         content for all
 
         -   Cost-free access enables access for all, regardless of economic
             means
 
-        -   Freedom from having to self-censor for fear of content consumption
```

Comment:
The section starting 'freedom from having to self-censor' is not clearly stated. I am unsure what self-censoring has to do with content consumption, I am unclear on the societal value of 'freedom from self-censorship' or why this group has an interest in it. I would argue that this line should be struck and we can add a major point under 'Interests of society' concerned with similar concepts instead.